packages/sdk/src/tables/base/table.ts

```diff
@@ -34,3 +38,5 @@ export default function Table<I, D>(config: { makeId: MakeId<I, D>; type: string
     type,
   };
 }
+
+export type Table = ReturnType<typeof Table>;
```

Comment:
i wonder if this matters:

`export type Table = ReturnType<typeof Table<I, D, S extends stores.Store<I, D>>;`